index.js

```diff
@@ -9,7 +9,7 @@ const pTimeout = require('p-timeout');
 const EventEmitter = require('events');
 
 class Queue extends EventEmitter {
-  constructor(mongoUrl, collectionName, waitDelay = 500, maxThreads = 1, prom = undefined, timeout = 30000) {
+  constructor(mongoUrl, collectionName, waitDelay = 500, maxThreads = 1, prom = undefined, timeout = 30000, autoretry = false) {
```

Comment:
Autorety should be job specific

Reply:
So when a create a job, I'll say wether it should be retried or not. Some jobs we won't want to auto retry